CHANGES

```diff
@@ -120,6 +115,123 @@ Tools
   Verbose Compat vernacular, since these warnings can now be silenced or
   turned into errors using "-w".
 
+Bugfixes
+
```

Comment:
@herbelin I don't know exactly what the intent of this list is. I did it a bit the same way as the ones for earlier versions. We should indeed discuss it at some point, but maybe we can do so after the beta. Note that the title of the bug is not always the description of the change. I think the list should list the changes, rather than the original symptom. But I'm not sure.